src/test/java/hudson/plugins/copyartifact/DownstreamBuildSelectorTest.java

```diff
@@ -31,7 +31,7 @@
 import java.util.Map;
 import java.util.Set;
 
-import hudson.plugins.copyartifact.testutils.CopyArtifactUtil;
+import hudson.plugins.copyartifact.testutils.CopyArtifactUtil;

```

Comment:
As far as I can see, this was the only line in this file with LF line ending. I've found 26 files altogether that dos2unix would change.